util/src/org/commcare/util/cli/EntityListSubscreen.java

```diff
@@ -132,6 +133,7 @@ public void prompt(PrintStream out) {
     @Override
     public boolean handleInputAndUpdateHost(String input, EntityScreen host) throws CommCareSessionException {
         if("action".equals(input) && mAction != null) {
+            // TODO PLM: handle multiple actions
             host.setPendingAction(mAction);
```

Comment:
@ctsims Not sure how to handle multiple actions in a detail for this specific context.